handlers/handlers.go

```diff
@@ -0,0 +1,63 @@
+package handlers
+
+import (
+	"dashboard/database"
+	"dashboard/model"
+	"database/sql"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+var db *sql.DB

```

Comment:
This is not needed

Reply:
if i remove this, then in all database functions ...db is undefined.